handlers/slots.go

```diff
@@ -162,8 +167,18 @@ func GetSlotsTableData(draw, start, length uint64, search string, searchForEmpty
 			searchBlocksQrys := []string{}
 			searchProposersQrys := []string{}
 
-			args = append(args, "%"+search+"%")
-			searchBlocksQrys = append(searchBlocksQrys, fmt.Sprintf(`(select slot from blocks where graffiti_text ilike $%d order by slot desc limit %d)`, len(args), searchLimit))
+			ilikeSearch := "%" + search + "%"
+			var graffiti [][]byte
+			statsQry := `SELECT DISTINCT(graffiti) FROM graffiti_stats WHERE graffiti_text ILIKE $1`
+			err = db.ReaderDb.SelectContext(ctx, &graffiti, statsQry, ilikeSearch)
```

Comment:
Question: What does `SelectContext` do (in contrast to just `Select`)?

Reply:
It just adds a context as a first parameter.
I have only ever seen this used to add a timeout to queries, in this case 30 seconds.

I now also notice that I accidentally left it at 30 minutes which I used for testing, this is not changed back to 30 seconds.

If you think that only the last query should check for a timeout I can quickly change it.
Maybe the one at the top if you don't search for anything should have one?

A 30 seconds timeout currently makes no sense anyway since we have a page timeout of 15s, I think it was this value here https://github.com/gobitfly/eth2-beaconchain-explorer/blob/40bd8cac004b1e726d8b84d5427a6d93c41df4a6/cmd/explorer/main.go#L620

but that's separate issue.